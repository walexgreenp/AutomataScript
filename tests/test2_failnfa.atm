PRINT: Basic test call;
PRINT: Create vaild AND invalid test;
PRINT: Should compile properly but should output some sort of fail on the users end;

// Passing test
Pass = "passing";
TEST: Pass << "passing";

// Failing test ("failing" not  "fail")
Fail = "failing";
TEST: Fail << "fail";
