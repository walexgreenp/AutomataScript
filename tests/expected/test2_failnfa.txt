------------------
* AutomataScript *

- Basic test call 
- Create vaild AND invalid test 
- Should compile properly but should output some sort of fail on the users end 
DELETEDELETETESTINGGITHUBCI
-----------------
